package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axelar-network/axelar-cgp-solidity",
-  "version": "6.3.0",
+  "version": "6.3.1",
```

Comment:
will need to cherry pick this commit to the release branch for the patch release. or fine with a minor version bump since we're adding L2 support